grades/models.py

```diff
@@ -141,6 +142,28 @@ def __str__(self):
         )
 
 
+class MicromastersCertificate(TimestampedModel):
```

Comment:
I'm confused about the name of this model. Will it only contain course certificates, or will it contain course and program (sometimes called "MicroMasters" certificates?